core/src/main/java/com/sequenceiq/cloudbreak/service/stack/flow/UpdateNodeCountValidator.java

```diff
@@ -80,7 +81,12 @@ public void validateServiceRoles(Stack stack, InstanceGroupAdjustmentV4Request i
                 .filter(e -> e.getName().equals(instanceGroup))
                 .findFirst();
         if (hostGroup.isPresent()) {
+            User creator = stack.getCreator();
+            String userCrn = creator.getUserCrn();
```

Comment:
Shouldn't we gather and check the user who initiated this scaling operation?

Reply:
nope